README.md

```diff
@@ -1,3 +1,105 @@
-# imgix-api-js
+<!-- ix-docs-ignore -->
 
-A JS library that wraps the imgix public API
+![imgix logo](https://assets.imgix.net/sdk-imgix-logo.svg)
+
+`imgix-management-js` is a JavaScript client library for making requests against the [imgix](https://www.imgix.com/) management API.
```

Comment:
I think we're treating "Management API" as a product name, so let's capitalize it.

```suggestion
`imgix-management-js` is a JavaScript client library for making requests against the [imgix](https://www.imgix.com/) Management API.
```